include/byte_order_generic_x86_64.h

```diff
@@ -88,35 +111,31 @@ static inline ulonglong uint6korr(const void *p)
 
 static inline ulonglong mi_uint5korr(const void *p)
 {
-  ulonglong a= *(uint32 *) p;
-  ulonglong b= *(4 + (uchar *) p);
-  ulonglong v= (a | (b << 32)) << 24;
+  ulonglong v= 0;
+  memcpy((uchar *) &v + 3, p, 5);
```

Comment:
An alternative would be `ulonglong v= uint5korr() << 24;`, but I wonder which variant is going to be faster?

Reply:
This won't be correct. We need here to swap first with last, second with pre-last and so on. Not just shift.